crates/nargo/src/constants.rs

```diff
@@ -1,23 +1,23 @@
 // Directories
 /// The directory for the `nargo contract` command output
-pub(crate) const CONTRACT_DIR: &str = "contract";
+pub const CONTRACT_DIR: &str = "contract";
```

Comment:
Why would those not be under target? They are product, which is reproducible... Would prefer to have one directory to delete (when cleaninig) instead of 3.